src/TTextEdit.cpp

```diff
@@ -1651,18 +1691,27 @@ void TTextEdit::resizeEvent(QResizeEvent* event)
 
 void TTextEdit::wheelEvent(QWheelEvent* e)
 {
-    int k = 3;
-    if (e->delta() < 0) {
-        mpConsole->scrollDown(abs(k));
-        e->accept();
-        return;
-    }
-    if (e->delta() > 0) {
+    const int k = 3;
+    int deltaY = e->angleDelta().y() / (8 * 15);
+    int deltaX = e->angleDelta().x() / (8 * 15);
+    bool handled = false;
+    if (deltaY < 0) {
+        mpConsole->scrollDown(k);
+        handled = true;
+    } else if (e->delta() > 0) {
```

Comment:
Scrolling left and right works for me, but scrolling to the left causes it to also scroll up at the same time.  I think that this should be `deltaY` instead of `e->delta()`.

Reply:
Thanks for spotting that. Please retest :)